v2.1/getting-started/openstack/upgrade.md

```diff
@@ -53,12 +53,9 @@ Stop the etcd process:
 
     systemctl stop etcd
 
-Then, download the tested binary (currently 2.0.11) and install it:
+Then upgrade:
```

Comment:
Shouldn't this cover removing the old hand-installed binary and installing the yum version too?